Source/Scene/BatchTable.js

```diff
@@ -169,6 +183,134 @@ define([
         return Number;
     }
 
+    function createOffsets(attributes, packFloats) {
+        var offsets = new Array(attributes.length);
+
+        var currentOffset = 0;
+        var attributesLength = attributes.length;
+        for (var i = 0; i < attributesLength; ++i) {
+            var attribute = attributes[i];
+            var componentDatatype = attribute.componentDatatype;
+
+            offsets[i] = currentOffset;
+
+            if (componentDatatype !== ComponentDatatype.UNSIGNED_BYTE && packFloats) {
```

Comment:
It would be cleaner to do this check outside of the loop to assign to a `stride` variable then here just do: `currentOffset  += stride;`